guacamole/src/main/webapp/app/rest/types/ConnectionGroup.js

```diff
@@ -129,6 +129,13 @@ angular.module('rest').factory('ConnectionGroup', [function defineConnectionGrou
         BALANCING : "BALANCING",
 
         /**
+         * The type string associated with balancing connection groups.
```

Comment:
This is the exact same message as used for the `BALANCING` type. This is probably a moot point, as I think adding a new `ROUNDROBIN` type should not be the solution here, but if it *were* this documentation would need to be updated to match what this type actually represents.